docs/mp/extensions/02_cdi_datasource-ucp.adoc

```diff
@@ -88,4 +88,4 @@ are parsed, and the remaining portion of each name is treated as a
 https://docs.oracle.com/javase/tutorial/javabeans/writing/properties.html[Java
 Bean property] of the
 https://docs.oracle.com/en/database/oracle/oracle-database/19/jjuar/oracle/ucp/jdbc/PoolDataSource.html[`oracle.ucp.jdbc.PoolDataSource`]
-class.
+class.
```

Comment:
This looks like a white space change. This should be displayed in the diff as "File renamed without changes"